mubench.reviewsite/tests/RunsControllerTest.php

```diff
@@ -40,7 +40,6 @@ function setUp()
             "detector" => "-d-",
             "project" => "-p-",
             "version" => "-v-",
```

Comment:
The above three fields are what is no longer sent, please remove them from the test.